zebra-scan/src/service.rs

```diff
@@ -138,8 +132,19 @@ impl Service<Request> for ScanService {
                 // TODO: send key_hashes and mpsc::Sender to scanner task, return mpsc::Receiver to caller
             }
 
-            Request::ClearResults(_key_hashes) => {
-                // TODO: clear results for these keys from db
+            Request::ClearResults(keys) => {
+                let mut db = self.db.clone();
+
+                return async move {
+                    // Delete the key from the database after either confirmation
```

Comment:
This comment seems incomplete.

Reply:
Yep, thank you, fixed in https://github.com/ZcashFoundation/zebra/pull/8219/commits/70e176e748f415cb6d977b7e09c93fd802a5b6f3.